fml/message_loop_impl.cc

```diff
@@ -118,8 +117,6 @@ void MessageLoopImpl::DoTerminate() {
 }
 
 void MessageLoopImpl::FlushTasks(FlushType type) {
-  TRACE_EVENT0("fml", "MessageLoop::FlushTasks");
```

Comment:
This is FML's only direct usage of tracing, and there is always at least one interesting event beneath it. Maybe we could guard this behind a flag, but it would be good to get rid of FML's reliance on the Dart VM here and instead have more descriptive events.

Reply:
Sounds good.